tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "forceConsistentCasingInFileNames": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
This is unfortunately a requirement for the time being until Material UI is released in full. If we don't use Material UI and we have the man power to style pages from scratch, I will turn this back on.